source/Ship.cpp

```diff
@@ -4016,7 +4098,10 @@ void Ship::DoGeneration()
 			{
 				Ship &ship = *it.second;
 				if(ship.HasDeployOrder())
+				{
 					DoRepair(ship.energy, energy, ship.attributes.Get("energy capacity"));
+					ship.SetState(Body::BodyState::LAUNCHING);
```

Comment:
```suggestion
					ship.SetState(BodyState::LAUNCHING);
```
(see one of my previous comments)